feature-libs/checkout/base/styles/components/steps/review/_review-submit.scss

```diff
@@ -0,0 +1,184 @@
+%cx-review-submit {
+  @include media-breakpoint-down(md) {
+    padding: 20px 0;
+  }
+
+  .cx-review {
+    padding: 0 0 1rem;
+  }
+  .cx-review-title {
+    text-transform: uppercase;
+    margin: 0 auto;
+    padding: 2.375rem 0 1.25rem 0;
+    @include type('3');
+    font-weight: var(--cx-font-weight-normal);
+  }
+
+  .cx-review-summary {
+    margin: 0;
+    background-color: var(--cx-color-background);
+    padding: 15px 0;
+
+    &-wrapper {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+      min-height: 100%;
+      word-break: break-all;
+    }
+
+    &-edit-step {
+      padding: 0 20px 20px;
+
+      @include media-breakpoint-down(md) {
+        padding: 10px 0 0;
+      }
+    }
+
+    .cx-review-payment-col {
+      @include media-breakpoint-up(lg) {
+        border-inline-end: 1px solid var(--cx-color-text);
+      }
+    }
+
+    .cx-review-summary-card {
+      display: flex;
+      justify-content: space-between;
+
+      @include media-breakpoint-down(md) {
+        background-color: theme-color('inverse');
+        border-style: solid;
+        border-width: 1px;
+        border-color: var(--cx-color-light);
+        min-height: auto;
+      }
+
+      &:not(:last-of-type) {
+        @include media-breakpoint-up(lg) {
+          border-bottom: 1px solid rgba(0, 0, 0, 0.5);
+        }
+      }
+
+      .cx-card-title {
+        @include type('4');
+        font-weight: bold;
```

Comment:
Please make sure to carry over the changes from here: https://github.com/SAP/spartacus/commit/3947c79e6af547ce27b828d3903c7cc9eaa1e390#diff-fc650a9686fe1201b0347cad376e1bbf83c4f660ef4105c6ca9c5e9b34899418